imap_processing/ena_maps/utils/naming.py

```diff
@@ -203,6 +365,50 @@ def ns_to_duration_months(ns: int) -> int:
     return int(months)
 
 
+def get_output_map_structure_from_descriptor_string(
```

Comment:
Should this be a method on the class?

```python
MapDescriptor.from_string(descriptor).to_empty_map()
```

Reply:
Yeah, that works! I'll change it here and in Ultra L2